### DISCUSSION TITLE
tranformations.unwrap() is very slow

### DISCUSSION THREAD

**Discussion by biodecoder5** [MDAnalysis/mdanalysis#4869] [Q&A] [answered]
Why is unwrap very slow. compared to trjconv -pbc whole i think it is around 300x or more slower.
am i using it wrong.
![Screenshot from 2024-12-27 22-57-05](https://github.com/user-attachments/assets/f60e9812-795b-4126-99e7-e9843bd66626)

900ms vs 5min

**Comment by orbeckst** [comment, marked as answer]
I am repeating my [comment from Discord #users](https://discord.com/channels/807348386012987462/807712221769367622/1325458411810193449) here:

> Unfortunately, unwrap is currently much slower than eg the implementation in GROMACS.
> There is a PR https://github.com/MDAnalysis/mdanalysis/pull/3169 open that tries to improve the speed but work there has stalled.

**Reply by biocodeit** [reply]
Thank you for the answer.
